package.json

```diff
@@ -498,7 +498,9 @@
     "workspaceSymbolProvider": "true"
   },
   "dependencies": {
+    "@types/fs-extra": "^5.0.0",
```

Comment:
Please move type def to devDependencies.

Reply:
Of course, sorry about that